src/components/accordion/Accordions.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion, { AccordionProps } from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import { styles } from '~/components/accordion/Accordion.styles'
+
+import { AccordionItem } from '~/types'
+import { Variant } from '@mui/material/styles/createTypography'
+
+interface Sx {
+  [key: string]: any
+}
+
+interface AccordionsProps
+  extends Omit<AccordionProps, 'onChange' | 'children'> {
+  items: AccordionItem[]
+  onChange: (value: number) => void
+  activeIndex: number[] | number | null
+  multiple?: boolean
+  icon?: ReactNode
+  sx?: Sx
+  titleVariant: Variant
+  descriptionVariant: Variant
+}
+
+const Accordions: FC<AccordionsProps> = ({
+  items,
+  onChange,
+  activeIndex,
+  multiple = false,
+  icon,
+  sx = styles,
+  titleVariant,
+  descriptionVariant,
+  ...props
+}) => {
+  const { t } = useTranslation()
+
+  const isMultiple = multiple && activeIndex instanceof Array
+
+  const accordionStyle = sx[icon ? 'withShowMoreIcon' : 'noShowMoreIcon']
+
+  return (
+    <Box sx={accordionStyle.root}>
+      {items.map((item, index) => (
+        <Accordion
+          data-testid={`${index}-${
+            isMultiple ? activeIndex.includes(index) : activeIndex === index
```

Comment:
add this to variable in map, active = isMultiple ? activeIndex.includes(index) : activeIndex === index

Reply:
i don't get what do you want, add field "active" to accordion items or what?

Reply:
isMultiple ? activeIndex.includes(index) : activeIndex === index
 you use this code a few times, move it to variable called active

Reply:
inside of map to make like  items.map((item, index) => { 
let active = false;
if(isMultiple) active = activeIndex.includes(index)
 }

Reply:
nope you dont need let here, in first coment try to use ternary 
const active = isMultiple ? activeIndex.includes(index) : activeIndex === index
return (...JSX)

Reply:
changed